src/aws/config.rs

```diff
@@ -79,6 +79,12 @@ pub struct Config {
     /// - profile config: `role_session_name`
     /// - default to `reqsign`.
     pub role_session_name: String,
+    /// `role_session_duration` value will be load from:
+    ///
+    /// - this field if it's `is_some`.
+    /// - profile config: `duration_seconds`
+    /// - default to `3600`.
+    pub role_session_duration: Option<i32>,
```

Comment:
I suggest keeping the same naming with AWS STS service to make it easier to understand.

Reply:
Thanks for the review. Refactored the variable name for consistency.